CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # tripwire_agent Cookbook CHANGELOG
 
+## 0.1.5 (TBD)
+
+Expanding axon's resources to support co-existing agents for TE, TLC, and IP360
```

Comment:
I'm not sure if TLC is supported in this scenario yet.  You might want to double-check that.